app/services/open_graph.rb

```diff
@@ -0,0 +1,80 @@
+# This class provides methods for getting structured metadata properties for
+# Open Graph usage
+
+# @note A wrapper around the metainspector gem
+class OpenGraph
+  delegate :meta, :meta_tags, :images, to: :page
+  delegate :favicon, to: :images
+
+  attr_accessor :page, :tags
```

Comment:
As we're setting instance variables, it looks like these should be `attr_reader`; no sense letting objects adjust these values.